satpy/readers/fci_l1c_fdhsi.py

```diff
@@ -7,6 +7,7 @@
 #
 #   Thomas Leppelt <thomas.leppelt@gmail.com>
 #   Sauli Joro <sauli.joro@icloud.com>
+#   Gerrit Holl <gerrit.holl@dwd.de>
```

Comment:
Could you update the copyright year as well. (2016-2019)

Reply:
Done in c7161c0